buildcraft_resources/assets/buildcraftfactory/compat/buildcraft/guide/en_us/block/mining_well.xml

```diff
@@ -0,0 +1,15 @@
+<chapter name="tile.miningWellBlock.name"/>
+<lore>
+On a quest to dig deeper, you start to research into some new machines and begin to assemble a machine that can automatically dig for you.
+It has one caveat, It can only dig straight down and does not dig bedrock.
+</lore>
+<no_lore>
+The Mining well is a machine that is used to automatically mine straight down all the way to bedrock.
+</no_lore>
+<chapter name="Information"/>
+It will mine directly downwards below where it is placed and will automatically out put the blocks it breaks into any inventories or pipes placed next to it.
```

Comment:
'out put' -> 'output'
maybe also mention it shoots them up in the air in case it fails to do that?